src/ape/types/signatures.py

```diff
@@ -22,6 +23,8 @@ class _Signature:
     v: int
     r: bytes
     s: bytes
+    _messageHash: HexBytes
```

Comment:
I think this might be better like this:
```suggestion
    _msg: Optional[SignableMessage] = None
```

Reply:
And add a note that this value can be added by downstream applications to cache this property for recovery purposes